### ISSUE TITLE
Test failures on i586

### ISSUE THREAD

**Issue by mnhauke** [andreax79/python-cstruct#15]
Some tests fail on i586:
```
[    5s] tests/test_define.py:47: AssertionError
[    5s] =========================== short test summary info ============================
[    5s] FAILED tests/test_alignment.py::test_foo1_sizeof - assert 3 == 7
[    5s] FAILED tests/test_alignment.py::test_foo2_sizeof - AssertionError: assert 16 ...
[    5s] FAILED tests/test_alignment.py::test_foo3_sizeof - AssertionError: assert 8 =...
[    5s] FAILED tests/test_alignment.py::test_foo5_sizeof - assert 3 == 7
[    5s] FAILED tests/test_alignment.py::test_foo10_sizeof - assert 3 == 7
[    5s] FAILED tests/test_define.py::test_sizeof - assert 8192 == 16384
[    5s] ========================= 6 failed, 41 passed in 0.17s =========================
```

Full buildlog:
[python-cstruct-buildlog-openSUSE-Tumbleweed.txt](https://github.com/andreax79/python-cstruct/files/9844981/python-cstruct-buildlog-openSUSE-Tumbleweed.txt)

**Comment by andreax79** [comment]
Expected padding values were wrong for 32bits arch.
Tests fixed in v 3.2

**Comment by mnhauke** [comment]
> Expected padding values were wrong for 32bits arch. Tests fixed in v 3.2

thanks, it now looks better but 1 test is still failing ...
```
[    4s] =================================== FAILURES ===================================
[    4s] _________________________________ test_sizeof __________________________________
[    4s] 
[    4s]     def test_sizeof():
[    4s]         assert sizeof('int') == 4
[    4s]         define('INIT_THREAD_SIZE', 2048 * sizeof('long'))
[    4s] >       assert cstruct.DEFINES['INIT_THREAD_SIZE'] == 16384
[    4s] E       assert 8192 == 16384
[    4s] 
[    4s] tests/test_define.py:48: AssertionError
[    4s] =========================== short test summary info ============================
[    4s] FAILED tests/test_define.py::test_sizeof - assert 8192 == 16384
[    4s] ========================= 1 failed, 49 passed in 0.17s =========================
```

**Comment by andreax79** [comment]
doh!

**Comment by andreax79** [comment]
Tests fixed in v 3.3
Now all tests on 32bits arch.
